Sources/MapboxMobileEvents/MMEEvent.m

```diff
@@ -443,7 +443,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
             return nil;
         }
         
-        _attributesStorage = [aDecoder decodeObjectOfClass:NSDictionary.class forKey:MMEEventAttributesKey];
+        _attributesStorage = [aDecoder decodeObjectOfClasses:[NSSet setWithObjects:NSArray.class, NSDictionary.class, NSString.class, NSNumber.class, NSNull.class, nil] forKey:MMEEventAttributesKey];
```

Comment:
Should we add some verification that the decoded object is a NSDictionary at the top level?